Jenkinsfile

```diff
@@ -0,0 +1,72 @@
+    [scratch: false, image: 'nexus3.edgexfoundry.org:10004/docker-device-rest-go:master', dockerfile: 'https://raw.githubusercontent.com/edgexfoundry/device-rest-go/master/Dockerfile'],
+    [scratch: false, image: 'nexus3.edgexfoundry.org:10004/docker-device-snmp-go:master', dockerfile: 'https://raw.githubusercontent.com/edgexfoundry/device-snmp-go/master/Dockerfile'],
+    [scratch: false, image: 'nexus3.edgexfoundry.org:10004/docker-device-virtual-go:master', dockerfile: 'https://raw.githubusercontent.com/edgexfoundry/device-virtual-go/master/Dockerfile'],
+    [scratch: false, image: 'nexus3.edgexfoundry.org:10004/docker-edgex-consul:master', dockerfile: 'https://raw.githubusercontent.com/edgexfoundry/docker-edgex-consul/master/Dockerfile'],
+    [scratch: false, image: 'nexus3.edgexfoundry.org:10004/docker-core-metadata-go:master', dockerfile: 'https://raw.githubusercontent.com/edgexfoundry/edgex-go/master/cmd/core-metadata/Dockerfile'],
+    [scratch: false, image: 'nexus3.edgexfoundry.org:10004/docker-core-data-go:master', dockerfile: 'https://raw.githubusercontent.com/edgexfoundry/edgex-go/master/cmd/core-data/Dockerfile'],
+    [scratch: false, image: 'nexus3.edgexfoundry.org:10004/docker-core-command-go:master', dockerfile: 'https://raw.githubusercontent.com/edgexfoundry/edgex-go/master/cmd/core-command/Dockerfile'],
+    [scratch: false, image: 'nexus3.edgexfoundry.org:10004/docker-support-notifications-go:master', dockerfile: 'https://raw.githubusercontent.com/edgexfoundry/edgex-go/master/cmd/support-notifications/Dockerfile'],
+    [scratch: false, image: 'nexus3.edgexfoundry.org:10004/docker-support-scheduler-go:master', dockerfile: 'https://raw.githubusercontent.com/edgexfoundry/edgex-go/master/cmd/support-scheduler/Dockerfile'],
+    [scratch: false, image: 'nexus3.edgexfoundry.org:10004/docker-sys-mgmt-agent-go:master', dockerfile: 'https://raw.githubusercontent.com/edgexfoundry/edgex-go/master/cmd/sys-mgmt-agent/Dockerfile'],
+    [scratch: false, image: 'nexus3.edgexfoundry.org:10004/docker-security-proxy-setup-go:master', dockerfile: 'https://raw.githubusercontent.com/edgexfoundry/edgex-go/master/cmd/security-proxy-setup/Dockerfile'],
+    [scratch: false, image: 'nexus3.edgexfoundry.org:10004/docker-security-secretstore-setup-go:master', dockerfile: 'https://raw.githubusercontent.com/edgexfoundry/edgex-go/master/cmd/security-secretstore-setup/Dockerfile'],
+    [scratch: false, image: 'nexus3.edgexfoundry.org:10004/docker-edgex-ui-go:master', dockerfile: 'https://raw.githubusercontent.com/edgexfoundry/edgex-ui-go/master/Dockerfile'],
+    [scratch: false, image: 'nexus3.edgexfoundry.org:10004/docker-security-bootstrapper-go:master', dockerfile: 'https://raw.githubusercontent.com/edgexfoundry/edgex-go/master/cmd/security-bootstrapper/Dockerfile']
+]
```

Comment:
Is there any way to make this data dynamic so that we don't have to maintain it statically?  i.e. querying the GitHub API edgexfoundry organization and iterating over the repos and querying for relevant information?  I'm sure you gave this some thought but would like to understand the complexities involved that forced you to implement a static list.

Reply:
Yeah, I gave a thought initially. But there are challenges: First of all, we need to have a list of nexus images (in use). Secondly, we need to find out the `Dockerfile` associated to each image. May be after we do some cleanup with image names & repositories, it would be easier to achieve.